.github/buildomat/jobs/package.sh

```diff
@@ -3,24 +3,11 @@
 #: name = "helios / package"
 #: variety = "basic"
 #: target = "helios-2.0"
-#: rust_toolchain = "1.72.1"
+#: rust_toolchain = "1.77.2"
```

Comment:
Is this supposed to stay in sync with `rust-toolchain.toml`? Can we read from that file directly instead? To be honest I didn't know this existed.

Reply:
Yes:

https://github.com/oxidecomputer/omicron/blob/d2ed4524f17ae025fb2e55956d7c82d377872a3e/rust-toolchain.toml#L1-L7

If it's not in sync it doesn't actually break anything, but the job runs a bit slower (Buildomat downloads 1.72.1 and then rustup's Cargo proxy sees you actually wanted 1.77.2 and downloads that).

Buildomat feature request: https://github.com/oxidecomputer/buildomat/issues/55